plugins/editor/views/editor.php

```diff
@@ -12,7 +12,10 @@
 $editor_file_input_name = $this->data('_editorFileInputName');
 
 foreach ($this->data('_EditorToolbar') as $button) {
-   $screenReaderMarkup = '<span class="sr-only">' . $button['attr']['title'] . '</span>';
+    $screenReaderMarkup = '';
+    if ($title = valr('attr.title', $button)) {
```

Comment:
It's weird to be setting the value of $title in the if statement. For clarity, I would put it outside.